react/src/pages/checks/ChecksPageNew.js

```diff
@@ -0,0 +1,244 @@
+import axios from 'axios';
```

Comment:
This file is not yet used. It is in progress refactor and just wanted it checked in for safe keeping.